content/swiftui/concepts/views/terms/lazyVStack/lazyVStack.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'LazyVStack'
+Description: 'A view that arranges its children in a line that grows vertically, creating items only as needed.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`LazyVStack`** has all the properties of a standard [`VStack`](https://www.codecademy.com/resources/docs/swiftui/views/vstack) but will not create items until it needs to be rendered on screen. This is useful when creating long lists and can allow an app to run smoother and faster.
```

Comment:
Taking another look at this, I feel the line from the meta-description should be included in this first paragraph like so:
```suggestion
The **`LazyVStack`** view arranges its elements in a line that grows vertically. It has all the properties of a standard [`VStack`](https://www.codecademy.com/resources/docs/swiftui/views/vstack) but it will not create items until it needs to be rendered on screen. This is useful when creating long lists and can allow an app to run smoother and faster.
```